src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -1874,23 +1206,77 @@ GlobeControls.prototype.setScale = function setScale(scale, pitch, isAnimated) {
  * @return {Promise} A promise that resolves when the next 'globe initilazed' event fires.
  */
 GlobeControls.prototype.setCameraTargetGeoPosition = function setCameraTargetGeoPosition(coordinates, isAnimated) {
-    return initPromise.then(() => {
-        isAnimated = isAnimated === undefined ? this.isAnimationEnabled() : isAnimated;
-        ctrl.targetGeoPosition = new C.EPSG_4326(coordinates.longitude, coordinates.latitude, 0);
-        updateAltitudeCoordinate(ctrl.targetGeoPosition, this._view.wgs84TileLayer);
-        const position = ctrl.targetGeoPosition.as('EPSG:4978').xyz();
-        position.range = coordinates.range;
-        return this.setCameraTargetPosition(position, isAnimated);
-    });
+    console.warn('setCameraTargetGeoPosition has been deprecated, use lookAtCoordinate');
+    return this.lookAtCoordinate(new Coordinates('EPSG:4326', coordinates.longitude, coordinates.latitude, 0), isAnimated);
+};
+
+/**
+ * Changes the center of the scene on screen to the specified coordinates
+ *
+ * @param      {Coordinates} coord specified coordinates
+ * @param      {boolean}    isAnimated  Indicates if change is animated
+ * @return     {Promise}    A promise that resolves when center is on specified coordinates
+ */
+GlobeControls.prototype.lookAtCoordinate = function _lookAtCoordinate(coord, isAnimated) {
```

Comment:
I would delete this function, rename the next one to `lookAtCoordinate`, and assume that if the first parameter is an instance of `Coordinates`,  you change it to `{ coord: params }`. I don't really like the `lookAtCoordinateAdvanced` name ;)